.github/workflows/build.yml

```diff
@@ -55,6 +55,12 @@ jobs:
         if: ${{ matrix.os == 'buildjet-4vcpu-ubuntu-2204-arm' || matrix.os == 'arm64-secure' }}
         run: |
           sudo apt install -y zstd
+       - name: install build dependencies
+        run: |
+          APT_INSTALL="sudo apt-get install -y golang"
+          # the following weird statement is here to speed up the happy path
+          # if the default server is responding -- we can skip apt update
+          $APT_INSTALL || { sudo apt update && $APT_INSTALL ; }
```

Comment:
Same for here; can we just add the [setup go action](https://github.com/actions/setup-go)?